docs/data/toolpad/getting-started/quickstart.md

```diff
@@ -0,0 +1,150 @@
+# Quickstart
+
+Get started building a Toolpad app!
+
+<p class="description">
+This page will guide you into creating your very first Toolpad application. We will create a "hello world" type of application that will ask for a name and print a greeting.
+</p>
+
+## Prerequisites
```

Comment:
Good to have this section here, we should probably add a "setup" page or something with more elaborate installation instructions

Reply:
I agree, as I mentioned in some other comment I think in the future we should add something like "Provisioning" section which explains in detail how to setup your own self-hosted version and not just run demo example